main.go

```diff
@@ -110,8 +149,27 @@ func main() {
 	var errorMsg string
 	if len(description) == 0 {
 		errorMsg = cfg.commentEmptyDescription
-	} else if len(description) <= len(template) {
-		errorMsg = cfg.commentTemplateNotFilled
+	} else {
+		sections := splitByHeaders(description)
+		errorMsg = ""
+		for _, section := range sections {
+			githubactions.Infof("Validating section: %s", section["header_text"])
+			if strings.Contains(section["header_text"], "Change Management"){
+				if len(section["content"]) < 5 {
+					errorMsg += fmt.Sprintln("**Change Management** section doesn't seem subtantial.") 
```

Comment:
```suggestion
					errorMsg += fmt.Sprintln("**Change Management** section is is required.") 
```

Copy was a little bit passive aggressive.

Also, I thought we were only going to require this for risky PRs?

Reply:
There's a typo. will fix